src/Codeception/Test/Metadata.php

```diff
@@ -19,71 +28,59 @@ class Metadata
         'incomplete' => null
     ];
 
+    /**
+     * @var array
+     */
     protected $current = [];
+    /**
+     * @var array
+     */
     protected $services = [];
-    protected $reports = [];
-
     /**
-     * @return mixed
+     * @var array
      */
+    protected $reports = [];
+
     public function getEnv()
     {
         return $this->params['env'];
     }
 
-    /**
-     * @return array
-     */
-    public function getGroups()
+    public function getGroups(): array
     {
         return array_unique($this->params['group']);
     }
 
-    /**
-     * @param mixed $groups
-     */
-    public function setGroups($groups)
+    public function setGroups($groups): void
     {
         $this->params['group'] = array_merge($this->params['group'], $groups);
     }
 
-    /**
-     * @return mixed
-     */
     public function getSkip()
     {
         return $this->params['skip'];
     }
 
-    /**
-     * @param mixed $skip
-     */
-    public function setSkip($skip)
+    public function setSkip($skip): void
     {
         $this->params['skip'] = $skip;
     }
 
-    /**
-     * @return mixed
-     */
     public function getIncomplete()
     {
         return $this->params['incomplete'];
     }
 
-    /**
-     * @param mixed $incomplete
-     */
-    public function setIncomplete($incomplete)
+    public function setIncomplete($incomplete): void
```

Comment:
I think that setSkip and setIncomplete are only called with strings.
and their getters return `?string`